pandas/tests/frame/test_reductions.py

```diff
@@ -1883,7 +1884,10 @@ def test_reduction_axis_none_returns_scalar(method, numeric_only):
         tm.assert_almost_equal(result, expected)
     else:
         expected = getattr(np, method)(np_arr, axis=None)
-        assert result == expected
+        if dtype == "float64":
```

Comment:
Why do you need this?

Reply:
Not needed, I removed it. I had been testing with pyarrow types and reductions of pyarrow types can have small floating pointing differences vs numpy.